bin/commands/runs.js

```diff
@@ -205,6 +207,12 @@ module.exports = function run(args, rawArgs) {
     markBlockStart('validateConfig');
     logger.debug("Started configs validation");
     return capabilityHelper.validate(bsConfig, args).then(function (cypressConfigFile) {
+      if(process.env.BROWSERSTACK_TEST_ACCESSIBILITY) {
+        setAccessibilityEventListeners(bsConfig);
+      }
+      if(process.env.BS_TESTOPS_BUILD_COMPLETED) {
+        // setEventListeners(bsConfig);
```

Comment:
Do we need this condition? If not needed, we should remove it.

Reply:
This is kept as reference for when we decide to fix event listeners for Cypress >= v12.15.1